woocommerce/woocommerce
# CYS: Add default patterns

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR adds some patterns that will be used from the CYS experience when the user doesn't have the tracking enabled. 

Futhermore, I fixed how the label of the category of a pattern is registered starting from the slug.

Closes #47402.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure that you have the last version of Gutenberg enabled
2. Make sure you have the WooCommerce beta tester plugin installed
3. Head over to Tools > WCA Test Helper > Features
4. Make sure you see the `pattern-toolkit-full-composability` on the list.
5. Enable it.
6. Visit `wp-admin/admin.php?page=wc-settings&tab=advanced&section=woocommerce_com`
7. Ensure that `Allow usage of WooCommerce to be tracked` isn't enabled.
8. Install [Transient Manager](https://wordpress.org/plugins/transients-manager/) plugin.
9. Visit `wp-admin/tools.php?page=transients-manager`
10. Search for `ptk_patterns`.
11. Ensure that the transient isn't set. If it is set, delete it.
13. Head over to WooCommerce > Home > Customize your store.
14. Click on Design Your Homepage.

For each category, ensure that all these patterns are visible:


![Default patterns (Woo plugin)](https://github.com/woocommerce/woocommerce/assets/4463174/5edbb38a-beb4-4d76-958e-d4bb03aa0885)

### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

CYS: Add default patterns.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
